src/ui/LiveSplit.tsx

```diff
@@ -367,6 +359,20 @@ export class LiveSplit extends React.Component<{}, State> {
         exportFile("layout.ls1l", JSON.stringify(layout, null, 4));
     }
 
+    public loadDefaultSplits() {
+        this.setState({
+            ...this.state,
+            timer: this.getTimerForDefaultRun(),
+        });
+    }
+
+    public loadDefaultLayout() {
+        this.setState({
+            ...this.state,
+            layout: Layout.defaultLayout(),
```

Comment:
Additionally, you are never disposing any objects. We can't yet integrate with JS's garbage collector. And because of that, the garbage collector can't clean up Rust's native memory. We basically don't get notified that some JS object, such as the old layout or the old run / timer aren't used anymore, so the underlying Rust object and its memory stay alive forever and thus you cause a memory leak. Check the other methods here for how we usually load layouts and runs. We have a bunch of helpers for disposing old objects.